war/src/main/js/filter-build-history.js

```diff
@@ -330,7 +331,7 @@ function checkRowCellOverflows(row) {
                 // build name (first field) a block element, forcing the details and controls to wrap
                 // onto the next row (creating a second row).
                 buildName.classList.add('block');
-                var wrap = blockWrap(buildDetails, buildControls);
+                wrap = blockWrap(buildDetails, buildControls);
```

Comment:
Ah yes, the intricacies of variable scope in JavaScript… :smile: I don't understand this but I trust you.